astropy/constants/constant.py

```diff
@@ -60,10 +60,10 @@ def wrapper(self, *args, **kwargs):
 
         # The wrapper applies to so many of the __ methods that it's easier to
         # just exclude the ones it doesn't apply to
-        exclude = set(['__new__', '__array_finalize__', '__array_wrap__',
+        exclude = {'__new__', '__array_finalize__', '__array_wrap__',
```

Comment:
While I like these kind of cleanups I would prefer if the indentation (for the following lines) would be changed as well.

Reply:
yeah this is a bit of a microbug / nice-to-have in pyupgrade -- I haven't gotten around to fixing it because I use `autopep8` / [this calling construct](https://github.com/asottile/add-trailing-comma#multi-line-method-invocation-style----why) which avoids 14-space / arbitrary-space indents